scripts/completion-tests/test-completion.sh

```diff
@@ -118,12 +118,16 @@ trap "GOT_FAILURE=1" ERR
 BASH4_IMAGE=completion-bash4
 
 echo;echo;
-docker build -t ${BASH4_IMAGE} - <<- EOF
-   FROM bash:4.4
-   RUN apk update && apk add bash-completion ca-certificates
-EOF
+DF=$(mktemp ./DF.XXXXX)
+
+echo "FROM bash:4.4
+RUN apk update && apk add bash-completion ca-certificates
+COPY ./ ${COMP_DIR}/
+" > $DF
+
+docker build -f $DF -t ${BASH4_IMAGE} ${COMP_DIR}
+rm -f $DF
```

Comment:
IMHO, this could be simplified, and the change could be smaller if we do something like this:

```
docker build -t ${BASH4_IMAGE} -f - ${COMP_DIR} <<- EOF
   FROM bash:4.4
   RUN apk update && apk add bash-completion ca-certificates
   COPY ./ ${COMP_DIR}/
EOF
```

the similar approach can be applied to the other changes in this PR.

Reply:
@pedjak I've simplified the change following your suggestion